spec.html

```diff
@@ -36262,8 +36282,16 @@ <h1>
               1. Perform ! CreateDataPropertyOrThrow(_A_, ! ToString(𝔽(_i_)), _capturedValue_).
               1. If the _i_<sup>th</sup> capture of _R_ was defined with a |GroupName|, then
                 1. Let _s_ be the CapturingGroupName of that |GroupName|.
-                1. Perform ! CreateDataPropertyOrThrow(_groups_, _s_, _capturedValue_).
-                1. Append _s_ to _groupNames_.
+                1. Let _isMatchedElsewhere_ be *false*.
+                1. For each integer _j_ such that _j_ &ne; _i_, _j_ &ge; 1, and _j_ &le; _n_, do
+                  1. If the _j_<sup>th</sup> capture of _R_ was defined with a |GroupName|, then
+                    1. Let _sj_ be the CapturingGroupName of that |GroupName|.
```

Comment:
I would not object to refactoring these aliases, e.g. _s_ → _groupName_ and _sj_ → _otherName_.